### PR TITLE
Lsig Fund Check

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
- Backend of account info added, fund check added

### CODE REVIEW

File: src/pages/Proposal.vue [hunk 1]
```diff
@@ -379,30 +371,44 @@ export default defineComponent({
...
-				}
-				if (typeof this.daoStore.govt_id === "undefined") {
-					this.error = TOKEN_NOT_FOUND;
-					return;
+				const response = await getAccountInfoByAddress(lsig.address());
+				let amount = null;
+				if (response) {
+					amount = response?.amount;
 				}
+				if (amount) return;
```
Comment (rmeena840) [inline src/pages/Proposal.vue:461]: There has to be min ammount that is needed by lsig.
@vuvoth Can you confirm this please?

**Review by rmeena840 — APPROVED** [review]
Please check comments

File: src/pages/Proposal.vue (hunk 1 above)
Comment (Megha-Dev-19) [reply src/pages/Proposal.vue:461]: yes @vuvoth, can you tell me about it?

Comment (vuvoth) [reply src/pages/Proposal.vue:461]: Yes, I think so and this should be number algo we use for transfer when we create proprosal @Megha-Dev-19. Btw we should check the gov token when transfer. Did we do it?

Comment (Megha-Dev-19) [reply src/pages/Proposal.vue:461]: Like I talked to you about it, I will create a new PR for dao fund lsig

Comment (Megha-Dev-19) [reply src/pages/Proposal.vue:461]: And I have added the check for gov token transfer

**Pull request merged**